README.md

```diff
@@ -0,0 +1,75 @@
+[![Build Status](https://travis-ci.org/kuzzleio/sdk-java.svg?branch=master)](https://travis-ci.org/kuzzleio/sdk-java)
+
+Official Kuzzle JAVA SDK
+======
+
+## About Kuzzle
+
+A backend software, self-hostable and ready to use to power modern apps.
+
+You can access the Kuzzle repository on [Github](https://github.com/kuzzleio/kuzzle)
+
+## SDK Documentation
+
+The complete SDK documentation is available [here](http://docs.kuzzle.io/sdk-reference/)
+
+## Protocol used
+
+The JAVA SDK implement the websocket protocol.
+
+## Build
+
+### Clone the depot
+
+```sh
+git clone --recursive git@github.com:kuzzleio/sdk-java.git
+cd sdk-java
+git submodule update --init --recursive
+make
+```
+
+You should know have the SDK in the `build/java/build/libs` directory.
```

Comment:
You should now

Reply:
That build path seems awfully complicated. Why can't we have the SDK made available directly under the `build/` directory?

Reply:
Because java is awfully complicated and automatically generates the java/build/libs directory to put the .jar's in it itself :/

Reply:
I'm somewhat not surprised :open_mouth: 

I won't block the PR over this, but could we move the built files to a more accessible directory once the process is finished?